File: app/src/main/java/com/example/mvvm_artichecture_sample/data/remote/apimodel/ReposnseModel.java
package com.example.mvvm_artichecture_sample.data.remote.apimodel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ReposnseModel {

    @SerializedName("code")
    @Expose
    private Integer code;
    @SerializedName("result")
    @Expose
    private List<CountryModel> mCountryModel = null;
    @SerializedName("extra")
    @Expose
    private List<Object> extra = null;

    public Integer getCode() {
        return code;
    }

    public void setCode(Integer code) {
        this.code = code;
    }

    public List<CountryModel> getCountryModel() {
        return mCountryModel;
    }

    public void setCountryModel(List<CountryModel> countryModel) {
        this.mCountryModel = countryModel;
    }

    public List<Object> getExtra() {
        return extra;
    }

    public void setExtra(List<Object> extra) {
        this.extra = extra;
    }
}

File: app/src/main/java/com/example/mvvm_artichecture_sample/data/remote/apimodel/ReposnseModel.kt
package com.example.mvvm_artichecture_sample.data.remote.apimodel

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ReposnseModel {

    @SerializedName("code")
    @Expose
    var code: Int? = null
    @SerializedName("result")
    @Expose
    var countryModel: List<CountryModel>? = null
    @SerializedName("extra")
    @Expose
    var extra: List<Any>? = null
}
